SU-HSDO/suhumsci
# Four Kitchens Release - Sprint 35

# READY FOR REVIEW

## Summary
- SHS-5068: Bug: Exposed filters in left sidebar bleed into main region
- SHS-5069: Bug: Status message after saving content does not have background color
- SHS-5007: Adjust cards to prevent tabbing to redundant links
- SHS-4903: CKEditor5: Anchor Link plugin
- SHS-5063: Implementation: Update link icon colors on postcards & in footer
- SHS-5079: Bug: Image resolution issue on Banner Image component

## Need Review By (Date)
10/02

## Urgency
medium

## Steps to Test

SHS-5068
- Visit /admin/structure/views/view/hs_default_courses/edit/list , enable the exposed forms block and save (see step 2 above)
- Visit /courses  and click on "Layout" in the local actions tab
- Go to the left sidebar, click on "Add Block". Search for "exposed" and add the "Default Courses: All List Exposed form" block
- Save the layout. You should see the filters on the left sidebar
- Ensure that the form fields do not go beyond the column (like in this screenshot):

<img width="574" alt="Screenshot 2023-09-19 at 11 52 54" src="https://github.com/SU-HSDO/suhumsci/assets/2343957/b73cab80-3124-4523-a384-cc872e7444c3">

SHS-5069
- Edit a node in Colorful or Traditional themes
- Note that the "saved" message is themed with a background color now:

![Screenshot from 2023-09-21 13-10-57](https://github.com/SU-HSDO/suhumsci/assets/2343957/8dd7bec2-21f5-42bd-b752-250c6b9b0bf5)

SHS-5007
- Check News, Events, People and Publications views and confirm that:
  - Items with images should have the images wrapped in a link tag that has `tabindex="-1" aria-hidden="true"` attributes
  - "Read More" links should have `tabindex="-1" aria-hidden="true"` attributes

SHS-4903
- Check that `anchor_link` module is installed in `admin/modules`
- Check that all editor toolbars in `admin/config/content/formats` have the `anchor_link` plugin enabled  (flag icon)
- Create a new page at `node/add/hs_basic_page`
- Add content and an anchor link
- Save page
- Check that adding the anchor to the new page url navigates to the correct section

SHS-5063
- View the site in Colorful theme. 
- Ensure the local footer is set to dark mode
- In all color schemes, make sure the lock, pdf and file icons show in the local footer as white with gray hover. 
- Visit a page with Vertical Links Cards and add a link to one, like here: http://hs-colorful.suhumsci.loc/components/collections-cards
- Ensure that links are white with gray hover
- Switch to view site in Traditional theme. 
- Visit a page with Vertical Links Cards and add a link to one, like here: `http://hs-traditional.suhumsci.loc/components/collections-cards`
- Ensure that links are black with dark gray hover

SHS-5079
- Add a banner image to a test page (the Full-width Region of a Flexible Page)
- Check that images are using the new styles (the largest new image preset is 3000x750px).

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
